src/models/UserModel.ts

```diff
@@ -1,30 +1,12 @@
-import {
-  DataTypes,
-  Model,
-  InferAttributes,
-  InferCreationAttributes,
-  Association,
-  NonAttribute,
-  CreationOptional
-} from 'sequelize'
+import { DataTypes, Model, CreationOptional } from 'sequelize'
 import { sequelize } from '../db/connection'
-import { Book } from './BookModel'
 
-export class User extends Model<
-  InferAttributes<User, { omit: 'books' }>,
-  InferCreationAttributes<User, { omit: 'books' }>
-> {
+export class User extends Model {
   declare id: CreationOptional<number>
   declare username: string
+  declare password: string
   declare email: string
-
-  declare createdAt: CreationOptional<Date>
-  declare updatedAt: CreationOptional<Date>
-
-  declare books?: NonAttribute<Book[]>
-  declare static associations: {
-    books: Association<User, Book>
-  }
+  declare role: string
```

Comment:
Look at this comment [here](https://github.com/CA-G12/wms-solution-inc/pull/4#discussion_r996421637)

we will need the same for the `role` ['admin', 'customer']